roles/neutron-data-network/tasks/dnsmasq.yml

```diff
@@ -3,6 +3,9 @@
   lineinfile: dest=/etc/default/dnsmasq regexp="^DNSMASQ_EXCEPT="
               line="DNSMASQ_EXCEPT=lo"
 
+- name: install resolvconf
+  apt: name=resolvconf state=present
```

Comment:
This should be added to the list of packages we install here: https://github.com/blueboxgroup/ursula/blob/master/roles/neutron-data-network/tasks/main.yml#L5-L9

By doing it in a list we gain some efficiencies in the task itself.